### PR TITLE
Add per-band options for rpc bus connection parallelism

### PR DESCRIPTION
**Pull request by koct9i**
- Set requst multiplexing parallelism in chunk client
  
- Add for RPC bus band config options for range of multiplexing parallelism

Motivation: required connection parallelism more depends on cluster topology.
It's easier to provide configuration on this level than fixing all possible initiators.

### CODE REVIEW

**Comment by maxim-babenko** [comment]
Shipped!

**Comment by maxim-babenko** [comment]
Let's run some tests just in case

File: yt/yt/core/bus/tcp/config.cpp [hunk 1]
```diff
@@ -15,6 +15,20 @@ void TMultiplexingBandConfig::Register(TRegistrar registrar)
...
+        .Default(DefaultMinMultiplexingParallelism);
+
+    registrar.Parameter("max_multiplexing_parallelism", &TThis::MaxMultiplexingParallelism)
+        .GreaterThanOrEqual(1)
+        .Default(DefaultMaxMultiplexingParallelism);
+
+    registrar.Postprocessor([] (TThis* config) {
+        THROW_ERROR_EXCEPTION_UNLESS(
+            config->MinMultiplexingParallelism <= config->MaxMultiplexingParallelism,
+            "min_multiplexing_parallelism exceeds max_multiplexing_parallelism");
```
Comment (maxim-babenko) [inline yt/yt/core/bus/tcp/config.cpp:30]: Let's place option names in double quotes

File: yt/yt/core/bus/public.h [hunk 2]
```diff
@@ -19,6 +19,9 @@ using TTosLevel = int;
 constexpr int DefaultTosLevel = 0;
 constexpr int BlackHoleTosLevel = -1;
 
+constexpr int DefaultMinMultiplexingParallelism = 1;
+constexpr int DefaultMaxMultiplexingParallelism = 1000;
```
Comment (maxim-babenko) [inline yt/yt/core/bus/public.h:23]: 1'000

File: yt/yt/core/rpc/channel.h [hunk 3]
```diff
@@ -78,6 +78,7 @@ struct TSendOptions
     bool GenerateAttachmentChecksums = true;
     bool RequestHeavy = false;
     EMultiplexingBand MultiplexingBand = EMultiplexingBand::Default;
+    // Parallelism is adjusted by per-brand configuration.
```
Comment (maxim-babenko) [inline yt/yt/core/rpc/channel.h:81]: per-band

**Review by maxim-babenko — CHANGES_REQUESTED** [review]

**Review by Kontakter — APPROVED** [review]
To rerun tests in internal CI

**Pull request closed without merging**